lexicons/app/bsky/administration/takeModerationAction.json

```diff
@@ -0,0 +1,30 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.administration.takeModerationAction",
```

Comment:
maybe just `moderate()`?

Reply:
My only concern would be that it becomes more concise but maybe trickier to think/talk about without referencing the "moderation action."  E.g. what would the name of the reversal method be, and what would be the "thing" that you're passing an id for to the reversal method?